README.md

```diff
@@ -41,12 +41,12 @@ The A2A Java SDK provides a Java server implementation of the [Agent2Agent (A2A)
 
 ```xml
 <dependency>
-    <groupId>io.a2a.sdk</groupId>
+    <groupId>io.github.a2asdk</groupId>
     <artifactId>a2a-java-sdk-client</artifactId>
```

Comment:
This client dependency isn't needed here. I'm going to submit a PR shortly to remove this and make some other updates to the instructions in the README.